flow/endpoint.yaml

```diff
@@ -91,3 +97,46 @@ components:
             - '/components/schemas/Device.Ipv6/properties/name'
             - '/components/schemas/Device.Bgpv4RouteRange/properties/name'
             - '/components/schemas/Device.Bgpv6RouteRange/properties/name'
+
+    Flow.Device.OneToOne:
+      description: >-
```

Comment:
I am ok with this if we know that device endpoint meshing is the most common scenario.
Otherwise, I would propose repurposing this model as `Flow.Device.Meshed` (to mean many-to-many) and `Flow.Device` would default to one-to-one mappning.